src/Service/Messenger.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Service;
+
+
+use App\Model\Message;
+
+class Messenger
+{
+    /**
+     * @var SenderInterface[]
+     */
+    protected $senders = [];
```

Comment:
Missing property type declaration;
Maybe this should be `private` instead or `protected`?
```suggestion
    protected array $senders = [];
```